quipucords/api/insights_report/serializers.py

```diff
@@ -27,6 +27,7 @@ class FactsSerializer(Serializer):
     last_discovered = fields.DateTimeField()
     qpc_server_version = fields.CharField(default=server_version)
     qpc_server_id = fields.CharField(default=get_server_id)
+    installed_products = fields.ListField(child=fields.DictField())
```

Comment:
Is this the correct place?

@bruno-fs mentioned `installed_products` will be in `system_profile`: https://gitlab.cee.redhat.com/insights-qe/iqe-foreman-rh-cloud-plugin/-/merge_requests/138#note_6358332

Satellite plugin also puts installed_products in system_profile.

Reply:
Hmm… I'll take a closer look at this.

Reply:
pls move it to `SystemProfileSerializer`

Reply:
Fixed, I think! Thanks for catching this.